java/spacewalk-java.spec

```diff
@@ -906,6 +906,14 @@ fi
 %{jardir}/spark-template-jade.jar
 %{jardir}/simpleclient*.jar
 %{jardir}/pgjdbc-ng.jar
+%{jardir}/java-saml-core.jar
+%{jardir}/java-saml.jar
+%{jardir}/joda-time.jar
+%{jardir}/stax-api.jar
+%{jardir}/stax2-api.jar
+%{jardir}/woodstox-core-asl.jar
+%{jardir}/xmlsec.jar
+
```

Comment:
Do we really need this new empty line? In fact I think we should remove the next one as well.